website/client/components/inventory/stable/index.vue

```diff
@@ -935,7 +935,20 @@
         }
       },
       async feedAction (petKey, foodKey) {
-        const result = await this.$store.dispatch('common:feed', {pet: petKey, food: foodKey});
+        let result;
+        try {
+          result = await this.$store.dispatch('common:feed', {pet: petKey, food: foodKey});
+        } catch (e) {
+          const errorMessage = e.message || e;
+
+          this.$store.dispatch('snackbars:add', {
+            title: 'Habitica',
+            text: errorMessage,
+            type: 'error',
+            timeout: true,
+          });
+        }
+
```

Comment:
I think these 3 lines should be inside the try block as well, or you should return from the catch block because otherwise when an error occurs `result` will be undefined and accessing `result.message` will throw another error

Reply:
@paglias , gotcha! good catch, I didn't think about that case. Updated